automated/android/noninteractive-tradefed/setup.sh

```diff
@@ -13,14 +13,12 @@ if [ -n "${ANDROID_VERSION}" ] && echo "${ANDROID_VERSION}" | grep -q  "aosp-and
     java_path="/usr/lib/jvm/java-17-openjdk-amd64/bin/java"
 fi
 
-PKG_DEPS="coreutils usbutils curl wget zip xz-utils python-lxml python-setuptools python-pexpect aapt lib32z1-dev libc6-dev-i386 lib32gcc1 libc6:i386 libstdc++6:i386 libgcc1:i386 zlib1g:i386 libncurses5:i386 python-dev python-protobuf protobuf-compiler python-virtualenv python-pip python-pexpect psmisc"
-
 dist_name
 case "${dist}" in
     ubuntu)
         dpkg --add-architecture i386
         apt-get update -q
-        install_deps "${PKG_DEPS} ${JDK}"
+        install_deps "${JDK} ${PYTHON38}"
```

Comment:
Where do the variable PYTHON38 come from?

Reply:
it's removed now.